monai/networks/utils.py

```diff
@@ -552,6 +555,80 @@ def save_state(src: torch.nn.Module | dict, path: PathLike, **kwargs):
     save_obj(obj=ckpt, path=path, **kwargs)
 
 
+def convert_to_onnx(
```

Comment:
With `convert_to_torchscript` we have the option to save to a file, can we add that here?

Reply:
updated the function so that it can be called with a filename. Converted onnx model will be saved to the file if provided.